### PR TITLE
COG-1368 Change default view of user list and add Search for users with search fields

### PR DESCRIPTION
**Pull request by RamuKammagani-osi**
#### Which User story does this relate to (link)?
- https://osi-cwds.atlassian.net/browse/COG-1368

#### Brief feature description
- Change Default view of User List to Search for users with multiple fields in it. 
- Include inactive Checkbox & Office filter dropdown has been moved(updated) according to the design.

### CODE REVIEW

**Comment by RamuKammagani-osi** [comment]
retest this please

File: app/javascript/sagas/getUserListSaga.js [hunk 1]
```diff
@@ -20,6 +20,6 @@ export function* countyUsersList(action) {
   }
 }
 
-export function* countyUsersListSaga() {
-  yield takeLatest(actionTypes.FETCH_USERS_API_CALL_REQUEST, countyUsersList)
+export function* SearchUserListSaga() {
```
Comment (kelfink) [inline app/javascript/sagas/getUserListSaga.js]: the functions should have lowercase initial letter

Comment (RamuKammagani-osi) [reply app/javascript/sagas/getUserListSaga.js]: Updated!

**Comment by RamuKammagani-osi** [comment]
retest this please

**Pull request merged**